lib/osf-components/addon/components/registries/read-only-contributors-list/template.hbs

```diff
@@ -0,0 +1,16 @@
+<div>
+    <ContributorList
+        @node={{this.node}}
+        @shouldTruncate={{false}}
+        @shouldLinkUsers={{true}}
+    />
+    <br>
```

Comment:
`<br>` makes sense semantically only when the line break is an essential/intrinsic part of the thing around it. My go-to examples are a multi-line street address or a poem.

In this case, it'd be better to put a local-class on the wrapping `div` with `display: flex;` (feels cleaner than adding another `div` in here just for formatting)